services/src/kilda-core/kilda-persistence-neo4j/src/test/java/org/openkilda/persistence/repositories/impl/Neo4jFlowMeterRepositoryTest.java

```diff
@@ -32,7 +30,6 @@
 import org.junit.Test;
 
 import java.util.Collection;
-import java.util.Optional;
 
 public class Neo4jFlowMeterRepositoryTest extends Neo4jBasedTest {
```

Comment:
Please, supply with a test(s) to cover the cases: 0 meters for a path, > 2 meters for a path, etc.

Reply:
will do!